### DISCUSSION TITLE
avr128db64 support

### DISCUSSION THREAD

**Discussion by concon** [lkankowski/arduino-multi-relay2#47] [Q&A]
Hi, what can i do for avr128db64 support ?

**Comment by lkankowski** [comment]
Well, I dont do active development on this project nor have such a device.
Probably all you need is to add configuration for your device in platformio.ini and make sure if used libraries have support for it.

**Comment by concon** [comment]
Thank you for your quick reply, I configured platformio.ini for avr128db64 but I get an error like this, 
" if (IS_VALID_DIGITAL_PIN(pin)) { " was not declared in this scope
Any help would be greatly appreciated.

Best regards,

**Reply by concon** [reply]
changed the processor architect to megaavr, it's ok

**Reply by lkankowski** [reply]
Glad you made it. Hope your project will be successful :)
